src/Microsoft.TemplateEngine.Edge/Settings/Scanner.cs

```diff
@@ -108,8 +114,10 @@ private void ScanForComponents(MountPointScanSource source)
                         // TODO: figure out what to do with probing path registration when components are not found.
                         // They need to be registered for dependent assemblies, not just when an assembly can be loaded.
                         // We'll need to figure out how to know when that is.
+#pragma warning disable CS0618 // Type or member is obsolete
                         _environmentSettings.SettingsLoader.Components.RegisterMany(typeList);
                         _environmentSettings.SettingsLoader.AddProbingPath(Path.GetDirectoryName(asm.Key));
```

Comment:
should we call ComponentManager.AddProbingPath here?

Reply:
Problem is, that `IComponentManager` doesn't have that interface...